google/cloud/forseti/scanner/scanners/base_scanner.py

```diff
@@ -64,15 +66,10 @@ def _output_results_to_db(self, violations):
             list: Violations that encountered an error during insert.
         """
         (inserted_row_count, violation_errors) = (0, [])
-        try:
-            vdao = violation_dao.ViolationDao(self.global_configs)
-            (inserted_row_count, violation_errors) = vdao.insert_violations(
-                violations,
-                snapshot_timestamp=self.snapshot_timestamp)
-        except db_errors.MySQLError as err:
-            LOGGER.error('Error importing violations to database: %s\n%s',
-                         err, violations)
 
+        violation_access = self.service_config[0].violation_access(
```

Comment:
This code is untested, I recommend ensuring there is a test using the sqlite db engine.

Reply:
```scanner_dao_test.test_save_violations()``` should provide the test coverage here, since the code here is just calling the violation access.  I have, however, updated the TODO to add unit test when violation error is handled, which would be interested to test.